src/components/shared/map/MapCenter.tsx

```diff
@@ -77,36 +87,25 @@ const MapCenter = () => {
         console.error('Error saving map center:', error);
       }
     } else {
-      console.log('Center or currentUser.pi_uid is null');
+      console.log('User not authenticated or center coordinates are null');
     }
-  };
+  };  
 
   const handleClickDialog = () => {
     setShowPopup(false);
   };
 
-  // define map boundaries
```

Comment:
Not sure why you're removing these map constraints as they are implemented fixes for the Task @ [https://sharing.clickup.com/9015444408/t/h/86bzcywyg/IXLEADH43N0OGKW](https://sharing.clickup.com/9015444408/t/h/86bzcywyg/IXLEADH43N0OGKW).